src/main/webapp/app/admin/audits/audits.component.ts

```diff
@@ -59,7 +56,7 @@ export class AuditsComponent implements OnInit {
             fromDate: this.fromDate, toDate: this.toDate,
         }).subscribe((res: HttpResponse<any>) => {
             this.audits = res.body;
-            this.links = this.parseLinks.parse(res.headers.get('link'));
+            this.links = parseLinks(res.headers.get('link'));
```

Comment:
I noticed that the audit entries are not shown on the UI, although I do not see any notable changes that could have caused it. Maybe it is not a side effect from this PR. But worth looking at.

Reply:
I think it wasn't caused by this PR, but I've fixed it.